### PR TITLE
#1894061: maven repo link is not correct on sdk deprecation notification when there are multiple replaces.

### PR DESCRIPTION
**Pull request by wangmingliang-ms**
[AB#1894061](https://dev.azure.com/mseng/a4d27ce2-a42d-4b71-8eef-78cee9a9728e/_workitems/edit/1894061): maven repo link is not correct on sdk deprecation notification when there are multiple replaces.

### CODE REVIEW

**Review by Flanker32 — DISMISSED** [review]
LGTM

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/java/com/microsoft/azure/toolkit/intellij/azuresdk/enforcer/AzureSdkEnforcer.java [hunk 1]
```diff
@@ -62,12 +63,15 @@ private static void warnDeprecatedLibs(@AzureTelemetry.Property List<? extends A
     private static String buildMessage(@Nonnull List<? extends AzureJavaSdkEntity> libs) {
         final String liPackages = libs.stream().map(l -> {
             if (StringUtils.isNotBlank(l.getReplace())) {
-                final String replacePackage = l.getReplace().trim();
+                final String[] replacePackages = l.getReplace().trim().split(",");
+                final String replaces = Arrays.stream(replacePackages)
+                    .map(p -> String.format("<a href='%s'>%s</a>", getMavenArtifactUrl(p.trim()), p.trim()))
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/java/com/microsoft/azure/toolkit/intellij/azuresdk/enforcer/AzureSdkEnforcer.java:68]: do we need to convert replace artifact to HTML list?

Comment (wangmingliang-ms) [reply PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/java/com/microsoft/azure/toolkit/intellij/azuresdk/enforcer/AzureSdkEnforcer.java:68]: yes

Comment (wangmingliang-ms) [reply PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/java/com/microsoft/azure/toolkit/intellij/azuresdk/enforcer/AzureSdkEnforcer.java:68]: no need. that will make it has 3 levels of list, it's ugly

**Review by Flanker32 — APPROVED** [review]
LGTM

**Pull request merged**